docs/visual-basic/language-reference/statements/if-then-else-statement.md

```diff
@@ -86,27 +94,35 @@ If condition Then [ statements ] [ Else [ elsestatements ] ]
 > [!TIP]
 >  The [Select...Case Statement](../../../visual-basic/language-reference/statements/select-case-statement.md) might be more useful when you evaluate a single expression that has several possible values.  
   
-## Single-Line Syntax  
- You can use the single-line syntax for short, simple tests. However, the multiple-line syntax provides more structure and flexibility and is usually easier to read, maintain, and debug.  
+### Single-Line Syntax  
+ You can use the single-line syntax for a single condition with code to execute if it's true. However, the multiple-line syntax provides more structure and flexibility and is easier to read, maintain, and debug.  
   
  What follows the `Then` keyword is examined to determine whether a statement is a single-line `If`. If anything other than a comment appears after `Then` on the same line, the statement is treated as a single-line `If` statement. If `Then` is absent, it must be the start of a multiple-line `If`...`Then`...`Else`.  
   
  In the single-line syntax, you can have multiple statements executed as the result of an `If`...`Then` decision. All statements must be on the same line and be separated by colons.  
-  
-## Example  
+
+## Multiline Syntax Example
```

Comment:
given that you're completely revisiting the topic, I'd also fix the headings to be sentence case